tests/modules/test_combineobjects.py

```diff
@@ -112,16 +142,29 @@ def test_one_object_first_image(
                 module.output_object.value = method
                 module.run(workspace)
                 assert (workspace.get_objects(method).segmented == segment).all()
+                
+        def test_one_object_first_image_volume(
+            self, objects_x_volume, module, workspace_volume, merge_methods
+        ):
+            # Test merge methods with one object in initial set for volumes
+            segment = numpy.zeros((10, 10, 10))
+            segment[2:4,2:4,2:4] = 1
+
+            objects_x_volume.segmented = segment
+
+            for method in merge_methods:
+                module.merge_method.value = method
+                module.output_object.value = method
+                module.run(workspace_volume)
+                assert (workspace_volume.get_objects(method).segmented == segment).all()
```

Comment:
Use a `numpy.testing` function to compare arrays.

Reply:
Done - now we test numpy arrays using `numpy.testing.assert_array_equal`